packages/keystone/file/file-types/excel.js

```diff
@@ -45,6 +48,10 @@ class ExcelParser {
         workbook.SheetNames.forEach((sheetName) => {
             const worksheet = workbook.Sheets[sheetName]
             const range = excel.utils.decode_range(worksheet['!ref'])
+            if (range.e.r > this.maxRowsCount) {
```

Comment:
What if `maxRowsCount` is undefined?

Reply:
I think this means there shouldn't be a limit. I'll add a check for the existence of `this.maxRowsCount`